OWNERS.md

```diff
@@ -46,6 +46,18 @@ Scope: Discoverability within Backstage, including the home page, information ar
 | Raghunandan Balachandran | Spotify      | Infinite Buck$ | [soapraj](http://github.com/soapraj)     | raghunandanb#1114 |
 | Bailey Brooks            | Spotify      | Infinite Buck$ | [bailey](http://github.com/bailey)       | \_bailey4         |
 
+### Search
+
+Team: @backstage/search-maintainers
+
+Scope: The Backstage Search plugin
+
+| Name                     | Organization | Team           | GitHub                                        | Discord           |
+| ------------------------ | ------------ | -------------- | --------------------------------------------- | ----------------- |
+| Emma Indal               | Spotify      | Infinite Buck$ | [emmaindal](http://github.com/emmaindal)      | emmaindal#7503    |
+| Raghunandan Balachandran | Spotify      | Infinite Buck$ | [soapraj](http://github.com/soapraj)          | raghunandanb#1114 |
+| Alex Lorenzi             | Spotify      | ProTean        | [AlexLorenzi](https://github.com/AlexLorenzi) | alexlorenzi       |
```

Comment:
Should be add the rest of ProTean here, or just leave me as the POC for now?

Reply:
Wither way works from my PoV, as long as this represents the group of people at Spotify that are responsible for the maintenance of this area.

Reply:
@alexlorenzi let's start here with you as POC and then add more people as needed